src/sxastarter/src/components/PageContent/ArticleList.tsx

```diff
@@ -111,11 +110,6 @@ export const ThreeColumn = (props: ArticleListComponentProps): JSX.Element => {
       id={id ? id : undefined}
     >
       <div className="container">
-        <div className="row">
-          <div className="col">
-            <div className="title display-6">{t('More News') || 'More News'}</div>
```

Comment:
Is this intended? Don't we need it on Financial?

Reply:
We are not using this variant anywhere in Financial site